hedera-node/hedera-app/src/xtest/java/contract/Erc721OperationsTest.java

```diff
@@ -122,34 +125,34 @@ void setUp() {
 
     @Test
     void recordedErc721OperationsResultInExpectedState() throws IOException {
-        // GIVEN:
+        // given:
         setupFakeStates();
         setupErc721InitcodeAndWellKnownAccounts();
         scaffoldingComponent.workingStateAccessor().setHederaState(scaffoldingComponent.hederaState());
 
-        // WHEN:
+        // when:
         handleAndCommit(CONTRACT_SERVICE.handlers().contractCreateHandler(), synthCreateTxn());
-        //        handleAndCommit(
-        //                        CONTRACT_SERVICE.handlers().contractCallHandler(),
-        //                synthApproveTxn(TREASURY, PARTY_ADDRESS, 2),
-        //                synthApproveTxn(TREASURY, PARTY_ADDRESS, 3),
-        //                synthSetApprovalForAll(TREASURY, OPERATOR_ADDRESS, true),
-        //                synthSafeTransferFrom(OPERATOR, TREASURY_ADDRESS, COUNTERPARTY_ADDRESS, 13),
-        //                synthSafeTransferFrom(PARTY, TREASURY_ADDRESS, COUNTERPARTY_ADDRESS, 3),
-        //                synthSafeTransferFrom(TREASURY, TREASURY_ADDRESS, PARTY_ADDRESS, 8),
-        //                synthSafeTransferFrom(COUNTERPARTY, COUNTERPARTY_ADDRESS, PARTY_ADDRESS, 3));
-
-        // THEN:
+        handleAndCommit(
+                CONTRACT_SERVICE.handlers().contractCallHandler(),
+                synthApproveTxn(TOKEN_TREASURY_ID, PARTY_ADDRESS, 2),
+                synthApproveTxn(TOKEN_TREASURY_ID, PARTY_ADDRESS, 3),
+                synthSetApprovalForAll(TOKEN_TREASURY_ID, OPERATOR_ADDRESS, true),
+                synthSafeTransferFrom(OPERATOR_ID, TOKEN_TREASURY_ADDRESS, COUNTERPARTY_ADDRESS, 13),
+                synthSafeTransferFrom(PARTY_ID, TOKEN_TREASURY_ADDRESS, COUNTERPARTY_ADDRESS, 3),
+                synthSafeTransferFrom(TOKEN_TREASURY_ID, TOKEN_TREASURY_ADDRESS, PARTY_ADDRESS, 8),
+                synthSafeTransferFrom(COUNTERPARTY_ID, COUNTERPARTY_ADDRESS, PARTY_ADDRESS, 3));
+
+        // then:
         assertExpectedContract();
         assertExpectedBytecode();
-        //       assertExpectedStorage();
+        assertExpectedStorage();
     }
 
     private void handleAndCommit(@NonNull final TransactionHandler handler, @NonNull final TransactionBody... txns) {
         for (final var txn : txns) {
             final var context = scaffoldingComponent.contextFactory().apply(txn);
             handler.handle(context);
-            scaffoldingComponent.stack().commit();
+            ((SavepointStackImpl) context.savepointStack()).commit();
```

Comment:
Do you know the reason why the interface does not include the `commit()` method?

Reply:
Not really. 🤔